src/AccountTile/AccountTile-styled.js

```diff
@@ -31,10 +31,14 @@ const StyledAccountTile = styled.div`
   transition: ${props => props.theme.transition};
   background-color: ${props =>
     props.open ? props.theme.palette.offWhite : props.theme.palette.white};
-  &:hover {
-    background-color: ${props => props.theme.palette.offWhite};
-    cursor: pointer;
-  }
+
+  ${props.clickable &&
```

Comment:
You'll have to use a function here to reference props, similar to what we're doing on ln 32.